test/unit/colorspace_spec.js

```diff
@@ -0,0 +1,106 @@
+/* Copyright 2017 Mozilla Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ColorSpace } from '../../src/core/colorspace';
+
+describe('colorspace', function () {
+  class Test {
+    constructor(decode, n) {
+      this.decode = decode;
+      this.n = n;
+    }
+  }
+  it('should be true if decode is not an array', function() {
+    let test1 = new Test('I am a string', 0);
+    let result1 = true;
+    let returnValue1 = ColorSpace.isDefaultDecode(test1.decode, test1.n);
```

Comment:
Considering what tests that you've currently added in the patch, I'm not sure why the code needs to be so verbose!?
Something along these lines ought to be perfectly sufficient:
```javascript
expect(ColorSpace.isDefaultDecode('string', 0)).toBeTruthy(); // or .toBeFalsy() where applicable
```
or, if that doesn't fit on *one* line (e.g. below when you're passing arrays as arguments):
```javascript
let someVariableName = [...];
expect(ColorSpace.isDefaultDecode(someVariableName, 0)).toBeTruthy();
```